fast64_internal/panels.py

```diff
@@ -0,0 +1,46 @@
+
+import bpy
+from .fast64_internal import *
+
+sm64GoalImport = 'Import' # Not in enum, separate UI option
```

Comment:
its directly related to panel stuff, i could instead split these panels between the sm64 and oot directories but i felt like it was nice to keep them together

Reply:
Right, of course fast64 currently has sm64 and oot stuff outside of the sm64 and oot stuff, why would it be properly organized. nvm then. Maybe this will be tackled eventually

Reply:
huh? i can move it now while i'm making it, i was just keeping panel stuff together so that the logic would stay aligned between oot and sm64. also `sm64GoalImport` could just be `goalImport` if oot ever gets importing

Reply:
I was just whining about the current state of the codebase dw